src/graph/helpers/create-default-properties.ts

```diff
@@ -35,6 +36,7 @@ export const createDefaultProperties = (params: DefaultProperties & { entityId:
 
   if (cover) {
     // add ID of cover "Image" to property "Cover"
+    assertValid(cover);
```

Comment:
Might be useful to pass a specific error message to `assertValid` to denote what the issue is. Would help consumers know which of their ids is invalid and why more easily.

For example this one might be something like `Invalid id for "cover". Here's why ...`

Reply:
👍 will do this in a follow up PR